swift-sdk/Internal/IterableAPIInternal.swift

```diff
@@ -600,7 +630,7 @@ final class IterableAPIInternal: NSObject, PushTrackerProtocol, AuthProvider {
         }
     }
     
-    private func updateSDKVersion() {
```

Comment:
I like `SDK` better. We should usually try to capitalize 3 or less letter abbreviations.

Reply:
Relegated to future discussion

Reply:
Still need to rename it to SDK. I think that is the style used by Apple. Look at their APIs. We should try to follow Apple guidelines as much as possible.